Makefile

```diff
@@ -98,13 +98,17 @@ ifneq ($(INSTALL_PREFIX), /usr/local)
 		sudo ln -sfn ${INSTALL_PREFIX}/lib/cmake/graphscope-analytical /usr/local/lib/cmake/graphscope-analytical; \
 	fi
 endif
-ifeq (${ENABLE_JAVA_SDK}, ON)
+
+ifeq ($(ENABLE_JAVA_SDK), ON)
```

Comment:
Since GAE has a seperate CMakeList, is that possible to move these GAE specific actions to the CMakeList and keep this Makefile clear?
- these job should configured to be done in the CMakeList.txt in GAE, in the target install?

Reply:
I think your idea is great! maybe I will refactor the GAE-Java installing stuff (in a manner as you suggest) in a another pr? Cause that will make this pr more clear.